### DISCUSSION TITLE
Add page_number in the pdf_content_list.json

### DISCUSSION THREAD

**Discussion by CathZheng** [magicpdf/Magic-PDF#245] [Q&A]
Hi, I'm trying to use the pdf_content_list.json for further processing. I want to use the page number as a reference back to the original pdf. Is there anyway I can have the page_number for each element in the json. Right now I only have something like:
{
        "type": "text",
        "text": "Hello",
        "text_level": 1
 }
I saw that in the preprocess files there is page_ID, how can I add it to the content list?

**Comment by myhloli** [comment]
![image](https://github.com/user-attachments/assets/b6557ecf-7ca8-4ea5-b647-a1b5fe87af34)
added in master.
